### DISCUSSION TITLE
Is the gateway-operator source available somewhere?

### DISCUSSION THREAD

**Discussion by ibotty** [Kong/gateway-operator-docs#29] [General]
I did not find any source for the gateway operator.  I assume it's also open source.

**Comment by mflendrich** [comment]
Hi @ibotty,

Kong Gateway Operator is currently available as a Docker image without sources, as described in our [docs](https://docs.konghq.com/gateway-operator/latest/).

We expect to release Gateway Operator as open source soon.

**Comment by mflendrich** [comment]
In response to @frankgreco's comment https://github.com/Kong/kong/discussions/12156#discussioncomment-7766287

> Is the plan to open source it?

Yes - we plan to open source Kong Gateway Operator in the foreseeable future. I cannot provide a definitive date but we're actively working on getting this done. :)

> What is the general approach to filing bugs, getting help, etc. for that project?

Use this repo (Kong/gateway-operator-docs).
